File: pinyougou-sellergoods-interface/src/main/java/com/xt/pinyougou/service/SellerService.java
package com.xt.pinyougou.service;

import com.baomidou.mybatisplus.core.metadata.IPage;
import com.baomidou.mybatisplus.extension.service.IService;
import com.xt.pinyougou.entity.Seller;

/**
 * <p>
 *  商家 服务类
 * </p>
 *
 * @author xt
 * @since 2020-01-07
 */
public interface SellerService extends IService<Seller> {

    IPage<Seller> selectPage(Integer currentPage, Integer pageNum, Seller seller);

    boolean updateStatus(String sellerId, String status);
}

File: pinyougou-manager-web/src/main/resources/static/service/sellerService.js
// 商家管理 服务
app.service('sellerService', function ($http) {

    // 分页条件查询
    this.findPage = function (currentPage, pageNum, entity) {
        return $http.post('/consumer/seller/page?currentPage=' + currentPage + '&pageNum=' + pageNum, entity);
    };

    // 详情
    this.findOne = function (id) {
        return $http.get('/consumer/seller/' + id);
    };

    // 保存
    this.updateStatus = function (sellerId, status) {
        return $http.get('/consumer/seller/updateStatus/' + sellerId + '/' + status);
    };

});